README.md

```diff
@@ -165,6 +165,7 @@ export interface Profile {
 - `racComparison`: Requested Authentication Context comparison type. Possible values are 'exact','minimum','maximum','better'. Default is 'exact'.
 
 - `forceAuthn`: if set to true, the initial SAML request from the service provider specifies that the IdP should force re-authentication of the user, even if they possess a valid session.
+- `passive`: if set to true, the initial SAML request from the service provider specifies that the IdP should prevent visible user interaction if possible.
```

Comment:
Based on the spec, "if possible" isn't true. The spec says the IdP "must not". Is there a different resource where you found that it obeys this less strict implementation?

Reply:
> Based on the spec, "if possible" isn't true. The spec says the IdP "must not". Is there a different resource where you found that it obeys this less strict implementation?

I can't say I have for SAML. I am happy to change this language.